modules/imaging_qc/php/imaging_qc.class.inc

```diff
@@ -232,36 +198,36 @@ class Imaging_QC extends \NDB_Menu_Filter
             "Missing"               => "Missing",
         ];
 
-        $this->fieldOptions
-            = [
-                'site'             => $siteList,
-                'project'          => $projectList2,
-                'subproject'       => $subprojectList2,
-                'scanType'         => $scan_types,
-                'mRIParameterForm' => [
-                    "Incomplete" => "Incomplete",
-                    "Complete"   => "Complete",
-                ],
-                'scanDoneInMRIPF'  => [
-                    "Yes" => "Yes",
-                    "No"  => "No",
-                ],
-                'tarchive'         => [
-                    "In DICOM" => "In DICOM Archives",
-                    "Missing"  => "Missing",
-                ],
-                'scanLocation'     => $scan_location,
-                'qCStatus'         => [
-                    "Pass" => "Pass",
-                    "Fail" => "Fail",
-                ],
-                'uploadedBy'       => $uploaderList,
-                'selected'         => [
-                    "True"  => "True",
-                    "False" => "False",
-                ],
-                'visitLabel'       => $visitList,
-            ];
+        $this->fieldOptions = [
+
+            'site'             => $this->getSiteOptions(false, []),
```

Comment:
Why is the array of getSiteOptions empty here? Or is it using `access_all_profiles` by default?

Is there no module permissions for imaging_qc? I don't know very well that module or what should be its behaviour in terms of permissions. @ridz1208, do you know?

Reply:
@cmadjar Rebased !

Reply:
And yes, getOptions have default values: 
` public function getSiteOptions(
        bool $numeric_keys = true,
        array $allsitesPermission = ['access_all_profiles'],
        bool $study_site = true
    )`